projects/plugins/jetpack/extensions/index.json

```diff
@@ -1,98 +1,109 @@
 {
-	"production": [
-		"ai-assistant",
-		"ai-assistant-plugin",
-		"blogging-prompt",
-		"business-hours",
-		"button",
-		"calendly",
-		"cookie-consent",
-		"contact-form",
-		"contact-form/salesforce-lead-form",
-		"contact-info",
-		"donations",
-		"eventbrite",
-		"gathering-tweetstorms",
-		"gif",
-		"google-calendar",
-		"image-compare",
-		"instagram-gallery",
-		"likes",
-		"mailchimp",
-		"map",
-		"markdown",
-		"opentable",
-		"payments",
-		"pinterest",
-		"podcast-player",
-		"premium-content",
-		"publicize",
-		"rating-star",
-		"recurring-payments",
-		"related-posts",
-		"repeat-visitor",
-		"send-a-message",
-		"whatsapp-button",
-		"seo",
-		"sharing",
-		"shortlinks",
-		"simple-payments",
-		"slideshow",
-		"social-previews",
-		"story",
-		"subscriptions",
-		"tiled-gallery",
-		"tock",
-		"videopress",
-		"wordads",
-		"payments-intro",
-		"post-publish-qr-post-panel",
-		"payment-buttons",
-		"site-editor-snackbars",
-		"videopress/video",
-		"ai-assistant-support"
-	],
-	"beta": [
-		"ai-chat",
-		"amazon",
-		"author-recommendation",
-		"conversation",
-		"dialogue",
-		"google-docs-embed",
-		"launchpad-save-modal",
-		"post-publish-promote-post-panel",
-		"recipe",
-		"v6-video-frame-poster",
-		"videopress/video-chapters",
-		"paywall",
-		"ai-assistant-form-support"
-	],
-	"experimental": [ "ai-image", "ai-paragraph" ],
-	"no-post-editor": [
-		"ai-assistant",
-		"business-hours",
-		"button",
-		"calendly",
-		"contact-form",
-		"contact-info",
-		"donations",
-		"eventbrite",
-		"gif",
-		"google-calendar",
-		"instagram-gallery",
-		"mailchimp",
-		"map",
-		"markdown",
-		"opentable",
-		"pinterest",
-		"rating-star",
-		"related-posts",
-		"repeat-visitor",
-		"simple-payments",
-		"slideshow",
-		"tiled-gallery",
-		"tock",
-		"videopress",
-		"wordads"
-	]
+	"production": {
+		"prod": [
```

Comment:
If we decide to gradually "unbundle" blocks, modifying the `production`/`beta`/etc. arrays might make it harder to keep both approaches simultaneously (i.e. both the main chunk of assets, and smaller custom bundles).